index.php

```diff
@@ -21,6 +21,28 @@
     return $this->renderer->render($response, '/hello.php', $args);
 });
 
+
+$app->get('/login', function ($request, $response, $args) {
+    return $this->renderer->render($response, '/login.php', $args);
+});
+
+$app->post('/login', function ($request, $response, $args) {
+    $body = $request->getParsedBody();
+
+    $userinfo=[];
+    $stmt = $this->database->prepare('select name, password from user where name = ?');
+    $stmt->execute(array($body['login']));
+    while ($row = $stmt->fetch()){
+        $userinfo = $row ;
+    }
+    foreach ($userinfo as $useri){
+        if ($userinfo['name']==$body['login'] && password_verify($body['login'],$userinfo['password'])){
```

Comment:
The name check isn't useful, as the SQL query ensure that you only have the user that match the form value.